src/common/programReplay.ts

```diff
@@ -15,6 +16,7 @@ const HIDE_VERSION = true;
 const DEFINED_ONLY = true;
 const INCLUDE_STACK_SCOPE = false;
 const HIDE_TEMP = true;
+const PUYA_SOURCEMAPS_REGEX_BLOCKLIST = [/[/\\]algopy[/\\][^/\\]+\.py$/];
```

Comment:
This would need to accomodate all potential frontend synthetic methods, instead I think this can be resolved in puya itself., by not outputting a location for synthetic methods